kie-server-parent/kie-server-controller/kie-server-controller-api/src/main/java/org/kie/server/controller/api/model/KieServerSetup.java

```diff
@@ -58,4 +63,19 @@ public void setServerConfig(KieServerConfig serverConfig) {
     public void setContainers(Set<KieContainerResource> containers) {
         this.containers = containers;
     }
+
+    public Set<Message> getMessages() {
+        if (messages == null) {
+            messages = new HashSet<>();
+        }
+        return messages;
+    }
+
+    public void setMessages(Set<Message> messages) {
+        this.messages = messages;
+    }
+
+    public boolean isRegistered() {
```

Comment:
this method name is bit misleading as it indicates some kind of registration and the only thing it does is searches for error messages. So maybe we should change the name of the method to reflect it better...?

Reply:
@mswiderski change it to hasNoErrors